x-pack/plugins/security_solution/public/alerts/containers/detection_engine/rules/use_rules.tsx

```diff
@@ -89,12 +89,14 @@ export const useRules = ({
       isSubscribed = false;
       abortCtrl.abort();
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [
     pagination.page,
     pagination.perPage,
     filterOptions.filter,
     filterOptions.sortField,
     filterOptions.sortOrder,
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Doing a `sort` on every render is probably not advisable. Might be worth scheduling work to improve. We could probably just wrap it in a `useMemo` if nothing else.